libloki/libloki-protocol.js

```diff
@@ -46,7 +46,7 @@
     }
   }
 
-  async function getPreKeyBundleForNumber(pubKey) {
+  async function getPreKeyBundleForContact(pubKey) {
```

Comment:
This still seems ambiguous IMO.
How about `getContactPreKeyBundle` ?

Reply:
Trying to keep consistent, the functions with `getContact...` are all for handling data that has come for a contact, while `...ForContact` are for handling data we have generated for our contacts

Reply:
Right right right right.
Should have read it better the first time! My bad.